src/traccuracy/metrics/_divisions.py

```diff
@@ -63,11 +64,20 @@ class DivisionMetrics(Metric):
             buffer between 0 and max_frame_buffer
     """
 
-    needs_one_to_one = True
-
     def __init__(self, max_frame_buffer=0):
         self.frame_buffer = max_frame_buffer
 
+    def _validate_matcher(self, matched: Matched) -> bool:
+        "Matcher must be one to one"
+        name = matched.matcher_info["name"]
+
+        if name == "IOUMatcher":
+            flag = matched.matcher_info["one_to_one"]
```

Comment:
Can you also check if IOU threshold is > .5? Because then the IOU matcher is implicitly one-to-one (and IIRC it is is more efficient to not actually set the flag, since I think adds an extra step of computation after the initial matching)